sample/TKTransferSamples.m

```diff
@@ -0,0 +1,215 @@
+          } onError:^(NSError *e) {
+              // something went wrong
+              @throw [NSException exceptionWithName:@"GetTokenException"
+                                             reason:[e localizedFailureReason]
+                                           userInfo:[e userInfo]];
+          }];
+    // redeemToken done snippet to include in docs
+    
+    // make sure it worked
+    [self runUntilTrue:^ {
+        return (transfer != nil);
+    }];
+}
+
+- (void)testCancelTransferToken {
+    TokenIOSync *tokenIOSync = [[self sdkBuilder] buildSync];
+    TKMemberSync *payerSync = [self createMember:tokenIOSync];
+    TKMember *payer = payerSync.async;
+    TKAccountSync *payerAccountSync = [payerSync linkAccounts:[self createBankAuthorization:payerSync]][0];
+    TKAccount *payerAccount = payerAccountSync.async;
+    Alias *payeeAlias = [self generateAlias];
+    [tokenIOSync createMember:payeeAlias];
+    NSString *refId = @"purchase:2017-11-01:28293336394ffby";
+    __block Token *transferToken = nil;
+    
+    // createTransferToken begin snippet to include in docs
+    TransferTokenBuilder *builder = [payer createTransferToken:100.0
+                                                      currency:@"EUR"];
+    builder.accountId = payerAccount.id;
+    builder.redeemerAlias = payeeAlias;
+    builder.descr = @"Book purchase";
+    builder.refId = refId;
+    
+    [builder executeAsync:^(Token *t){
+            [payer endorseToken:t
+                        withKey:Key_Level_Standard
+                      onSuccess:^(TokenOperationResult *result) {
+                          transferToken = result.token;
+                      } onError:^(NSError *e) {
+                          // something went wrong
+                          @throw [NSException exceptionWithName:@"EndorseException"
+                                                         reason:[e localizedFailureReason]
+                                                       userInfo:[e userInfo]];
+                      }];
+    }   onError:^(NSError *e) {
+        // Something went wrong. (We don't just build a structure; we also
+        // upload it to Token cloud. So things can go wrong.)
+        @throw [NSException exceptionWithName:@"BuilderExecuteException"
+                                       reason:[e localizedFailureReason]
+                                     userInfo:[e userInfo]];
+    }];
+    
+    [self runUntilTrue:^ {
+        return (transferToken != nil);
+    }];
+    
+    // cancelToken begin snippet to include in docs
+    [payer cancelToken:transferToken
+             onSuccess:^(TokenOperationResult *result) {
+                 // token now has more signatures; in this case, at least
+                 // one is a cancellation signature
+                 transferToken = result.token;
+             } onError: ^(NSError *e) {
+                 // Something went wrong
+             }];
+    // cancelToken done snippet to include in docs
```

Comment:
Do you want a runUntilTrue here? (Be consistent)

Reply:
Done.